docs/core/tools/global-tools-how-to-create.md

```diff
@@ -0,0 +1,226 @@
+
+```csharp
+// Check if a message was provided
+if (string.IsNullOrEmpty(messageArg.Value))
+{
+    if (!app.IsShowingInformation)
+        app.ShowHelp();
+
+    return;
+}
+```
+
+If the check above succeeds, then the message argument was not passed and the application will quit. To help the user know that they need to provide the `message` argument, `app.ShowHelp()` is called. However, the `app.IsShowingInformation` will be set to `true` when the `help` or `version` argument options are passed, so we don't want to display the help information twice. 
+
+### Create the bot
+
+Next, add a new method named `ShowBot` that accepts a string parameter. This method will print out the message and the ascii bot.
+
+```csharp
+static void ShowBot(string message)
+{
+    string bot = $"\n        {message}";
+    bot += @"
+    __________________
+                      \
+                       \
+                          ....
+                          ....'
+                           ....
+                        ..........
+                    .............'..'..
+                 ................'..'.....
+               .......'..........'..'..'....
+              ........'..........'..'..'.....
+             .'....'..'..........'..'.......'.
+             .'..................'...   ......
+             .  ......'.........         .....
+             .                           ......
+            ..    #            ##        ......
+           ....       .                 .......
+           ......  .......          ............
+            ................  ......................
+            ........................'................
+           ......................'..'......    .......
+        .........................'..'.....       .......
+     ........    ..'.............'..'....      ..........
+   ..'..'...      ...............'.......      ..........
+  ...'......     ...... ..........  ......         .......
+ ...........   .......              ........        ......
+.......        '...'.'.              '.'.'.'         ....
+.......       .....'..               ..'.....
+   ..       ..........               ..'........
+          ............               ..............
+         .............               '..............
+        ...........'..              .'.'............
+       ...............              .'.'.............
+      .............'..               ..'..'...........
+      ...............                 .'..............
+       .........                        ..............
+        .....
+";
+    Console.WriteLine(bot);
+}
+```
+
+Next, back in the `static void Main(string[] args)` method, add a call `ShowBot` as the last line of the method, passing in the message argument value.
+
+```csharp
+ShowBot(messageArg.Value);
+```
+
+### Test the tool
+
+You can run the project and see the output. Try these variations of the command:
+
+```csharp
```

Comment:
It would be super useful to talk about the -- here or link to something covering it. Most people do not know how that works.